helm/ffc-pay-statement-generator/values.yaml

```diff
@@ -40,7 +40,10 @@ container:
   storageContainer: statements
   storageFolder: outbound
   statementVersion: v1
-  statementReceiverEndpoint: http://ffc-pay-statement-receiver.ffc-pay
+  statementReceiverEndpoint: https://ffc-pay-statement-receiver
```

Comment:
Original is the endpoint for the internal cluster network.  Requests will originate from CRM, not a Kubernetes pod so need to update to support external gateway.

This would first be `https` and also need to be dynamic as the subdomain changes in each environment.  The `ConfigMap` handles mapping the dynamic values.

Incidentally the current approach would not work beyond sandpit as all namespaces in higher environments have suffixes, eg `ffc-pay-dev` so even if this was a link for AKS services, would still need similar changes in the `ConfigMap`.